Writing September/October newsletter (#35)
OBF/newsletter: newsletters/2023-09.md

---
tags: obf
---

_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/FIX THIS LINK)._

_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._

# OBF Newsletter September 2023

> TL;DR: In this newsletter we announce the OBF's upcoming public board meeting soon; provide a summary of this year's BOSC in Lyonl; share news from the OBF event fellowship; and announce that we are looking for someone to support us in being the organisation-admin for the 2024 Google Summer of Code. 

Reviewer: ```suggestion
TL;DR: In this newsletter we announce the OBF's upcoming public board meeting; provide a summary of BOSC 2023 in Lyon; share news about the OBF event fellowship; and announce an opening for an OBF admin for the 2024 Google Summer of Code. 
```

In longer form: 

## OBF News

### Upcoming public board meeting

> The OBF is having it's next public board meeting at XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog. 

Reviewer: ```suggestion
The OBF is having its next public board meeting on XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog [add link]. 
```

### BOSC news

> This year we had the 24th annual Bioinformatics Open Source Conference, which took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceeded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants. 

Reviewer: ```suggestion
The 24th annual Bioinformatics Open Source Conference (BOSC 2023) took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants. 
```

> In addition to XXX talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_. 

Reviewer: ```suggestion
In addition to 43 talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_. 
```

You can [read the full report about the conference on the OBF blog](https://www.open-bio.org/2023/08/14/bosc-2023-report/).

### OBF Travel Fellowships

Next round: December 1, 2023
Last round: August 1, 2023

> The ongoing application period for the OBF Event Fellowship lasts until December 1, 2023. So there is still some [time to get your application submitted](https://docs.google.com/forms/d/e/1FAIpQLScMayX_XwTLxc8MiYgQOHIILJNjxF0gL75TUsbaO17h5UuUeA/viewform).

Reviewer: ```suggestion
The current application period for the OBF Event Fellowship lasts until December 1, 2023. So there is still some [time to get your application submitted](https://docs.google.com/forms/d/e/1FAIpQLScMayX_XwTLxc8MiYgQOHIILJNjxF0gL75TUsbaO17h5UuUeA/viewform).
```

> These fellowship can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).

Reviewer: ```suggestion
These fellowships can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).
```

### Google Summer of Code

> We are looking for a motivated person who would like to volunteer as the organisation admin to allow the OBF to participate in the 2024 Google Summer of Code. As an organisation we have been providing an umbrella project for various open source bioinformatics projects – including non-member projects – since 2010. In 2023 we sat out one round, as we did not find someone with the time to volunteer as GSoC admin. 

Reviewer: ```suggestion
We are looking for a motivated person who would like to volunteer as the organisation admin to enable the OBF to participate in the 2024 Google Summer of Code, as we have in previous summers. As an organisation we have been providing an umbrella project for various open source bioinformatics projects – including non-member projects – since 2010. In 2023 we sat out one round, as we did not find someone with the time to volunteer as GSoC admin. 
```

> If you are interested to fill that role, please get in touch with us at XXX.

Reviewer: ```suggestion
If you are interested in filling that role, please get in touch with us at XXX.
```

### ISCBacademy recording

> On March 14, the OBF hosted the _ISCBacademy_ where Hannah Wei – co-founder of the Patient-Led Research Collaborative – gave a talk on _"Re-Thinking the Patient’s Role in a Learning Health System: Lessons from the Patient-Led Research Collaborative"_. The [recording of the talk is now on YouTube](https://www.youtube.com/watch?v=M2vAotWKd_Q&feature=youtu.be).

Reviewer: ```suggestion
On March 14, 2023, the OBF hosted an _ISCBacademy webinar_ in which Hannah Wei – co-founder of the Patient-Led Research Collaborative – gave a talk on _"Re-Thinking the Patient’s Role in a Learning Health System: Lessons from the Patient-Led Research Collaborative"_. The [recording of the talk is now on YouTube](https://www.youtube.com/watch?v=M2vAotWKd_Q&feature=youtu.be).

Our next ISCBacademy webinar on October 3 will be jointly sponsored with the Bio-Ontologies COSI. Sierra Moxon will speak about "LinkML: an open data modeling framework, grounded with ontologies". Free registration at https://www.iscb.org/iscbacademy.
```

## Community updates

### Save the date: CWL Conference 2024

> The 2024 CWL Conference will be the week of May 13th, 2024 in the Netherlands. More details to follow.

Reviewer: ```suggestion
The 2024 Common Workflow Language (CWL) Conference will be the week of May 13th, 2024 in the Netherlands. More details to follow.
```

## Interesting links

### Your content needed for the next issue!

> Tell us about the things that catch your attention in the open source bioinformatics world! If you have an exciting project update, request for feedback, or interesting link, [feel free to share it with us on GitHub](https://github.com/OBF/newsletter/issues/FIX THIS LINK), or contact Geraldine Van der Auwera (Twitter: [@VdaGeraldine](https://twitter.com/VdaGeraldine), Email: [geraldinevdauwera@gmail.com](mailto:geraldinevdauwera@gmail.com)) if you're not sure whether or not your content is suitable for our newsletter. We're looking for content that's primarily open source / science related.

Reviewer: replace Geraldine with Bastian?

Author: done, thanks!

---
tags: obf
---

_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/FIX THIS LINK)._

_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._

# OBF Newsletter September 2023

> TL;DR: In this newsletter we announce the OBF's upcoming public board meeting; provide a summary of BOSC 2023 in Lyon; share news about the OBF event fellowship; and announce an opening for an OBF admin for the 2024 Google Summer of Code. 

Reviewer: ```suggestion
TL;DR: In this newsletter we provide a summary of BOSC 2023 in Lyon; share news about the OBF event fellowship; and announce an opening for an OBF admin for the 2024 Google Summer of Code. 
```

In longer form: 

## OBF News

### Upcoming public board meeting

> The OBF is having its next public board meeting on XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog [add link]. 

Reviewer: ```suggestion
The OBF will hold its next public board meeting some time this fall (date TBD). When the date is determined, we will announce it on the [OBF Blog](https://www.open-bio.org/blog/) and in social media, as well as email to OBF members.
```

### BOSC news

The 24th annual Bioinformatics Open Source Conference (BOSC 2023) took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants. 

In addition to 43 talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_. 

> You can [read the full report about the conference on the OBF blog](https://www.open-bio.org/2023/08/14/bosc-2023-report/).

Reviewer: Add:

For those who like to plan ahead, BOSC 2024 will take place between July 12-16, 2024, as part of [ISMB 2024](https://www.iscb.org/ismb2024) in Montréal, Québec, Canada.

> ### OBF Travel Fellowships

Reviewer: ```suggestion
### OBF Event Fellowships
```

Next round: December 1, 2023
Last round: August 1, 2023

The current application period for the OBF Event Fellowship lasts until December 1, 2023. So there is still some [time to get your application submitted](https://docs.google.com/forms/d/e/1FAIpQLScMayX_XwTLxc8MiYgQOHIILJNjxF0gL75TUsbaO17h5UuUeA/viewform).

> These fellowships can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).

Reviewer: Add

## Sponsors needed!

OBF Event Fellowships, which seek to increase diverse representation at relevant events, are funded by sponsorships. Most of these come through [BOSC](https://www.open-bio.org/events/sponsors/), but we also welcome donations to the OBF. Please contact us at [] if you are interested in discussing sponsorship possibilities.

Author: Added as well :) Do we have someone that should be the point person to link to?

Reviewer: i don't know. we need someone in OBF to take responsibility for that. (not me!)

Reviewer 2: Could default to "Please contact the board if ..." (need not be a email link), but I'm inclined to drop this text for now. We don't have anything setup to thank non-BOSC sponsors beyond a one off blog post like we had for the NESCent funds.

Reviewer: ok, we can take this out, but this is something we need to work on.

Author: Okay, I'll remove that part!

Our next ISCBacademy webinar on October 3 will be jointly sponsored with the Bio-Ontologies COSI. Sierra Moxon will speak about "LinkML: an open data modeling framework, grounded with ontologies". Free registration at https://www.iscb.org/iscbacademy.

## Community updates

### Save the date: CWL Conference 2024

The 2024 Common Workflow Language (CWL) Conference will be the week of May 13th, 2024 in the Netherlands. More details to follow.

### iCn3D received updates at CoFest

During CoFest 2023, the [iCn3D viewer](https://www.ncbi.nlm.nih.gov/Structure/icn3d/) was updated to include two new features: It can now show isoforms and exons as tracks and users can also predict structures from sequences using ESMFold directly in iCn3D. You can learn more about it [on the project's GitHub page](https://github.com/ncbi/icn3d).

> ## Interesting links

Reviewer: remove this section since it's empty?
